packages/table/src/FzTable.vue

```diff
@@ -0,0 +1,277 @@
+    <div
+      v-if="actions"
+      :class="['fz__table__header--actions', headerStaticClasses, headerClasses]">
+      Azioni
+    </div>
+    <template v-if="value && value.length" v-for="(row, index) in value">
+      <div
+        :aria-rowindex="index"
+        :class="[bodyStaticClasses, bodyClasses, getBodyClasses(column)]"
+        role="cell"
+        v-for="column in columns">
+        {{ row[column.props.field?.toLowerCase() || column.props.header.toLowerCase()] }}
+      </div>
+      <div
+        v-if="actions"
+        :class="[bodyStaticClasses, bodyClasses]">
+        <FzIconDropdown :actions="actions.items"></FzIconDropdown>
+      </div>
+    </template>
+    <div :class="['fz__table__empty', 'h-full mt-80', 'justify-self-center', colspan]" v-else>
+      {{ placeholder ?? "No data available" }}
+    </div>
+  </div>
+  
+</div>
+<div class="fz__table__footer w-full flex flex-row justify-end m-8" v-if="pages && value && value.length">
+  <div class="fz__table__pagination flex flex-row justify-between items-center gap-8">
+    <FzButton
+      @click="activePage = 0"
+      :variant="activePage === 0 ? 'primary' : 'secondary'"
+      size="sm">1</FzButton>
+    <div class="fz__pagination__separator" v-if="activePage - pageInterval - 1 > 0">...</div>
+    <FzButton
+      v-for="page in centerPageList"
+      @click="activePage = page"
+      :variant="activePage === page ? 'primary' : 'secondary'"
+      size="sm">
+        {{ page + 1 }}
+      </FzButton>
+    <div class="fz__pagination__separator" v-if="pages - activePage - pageInterval - 2 > 0">...</div>
+    <FzButton
+      @click="activePage = pages - 1"
+      :variant="activePage === pages - 1 ? 'primary' : 'secondary'"
+      size="sm">{{pages}}</FzButton>
+  </div>
+</div>
+</template>
+
+<style scoped>
+.fz__table .grid.grid-cols-1 {
+  grid-template-columns: repeat(1, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-2 {
+  grid-template-columns: repeat(2, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-3 {
+  grid-template-columns: repeat(3, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-4 {
+  grid-template-columns: repeat(4, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-5 {
+  grid-template-columns: repeat(5, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-6 {
+  grid-template-columns: repeat(6, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-7 {
+  grid-template-columns: repeat(7, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-8 {
+  grid-template-columns: repeat(8, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-9 {
+  grid-template-columns: repeat(9, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-10 {
+  grid-template-columns: repeat(10, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-11 {
+  grid-template-columns: repeat(11, minmax(min-content, 1fr))
+}
+.fz__table .grid.grid-cols-12 {
+  grid-template-columns: repeat(12, minmax(min-content, 1fr))
+}
```

Comment:
This is very verbose, you could just use a dynamic style with the repeat number interpolated

Reply:
nope. Sadly, tailwind would not compile those classes

Reply:
This is by design in tailwind, and it is in fact the main drawback of using css grid in my opinion

Reply:
unless you mean we go with custom interpolated `:style` binding, which is possible. I don't like it but it is probably better than this

Reply:
why does tailwind not compile these classes? They are in the documentation https://tailwindcss.com/docs/grid-template-columns

Reply:
yes, I meant using a custom js in css. It's the only doable way afaik

Reply:
@Crish15 I meant that if you interpolate names like grid-col-{variable} tailwind JIT will not work. Of course if we bypass that and go with :style, that's fine. Will do that I think, this honestly just sucks

Reply:
> @Crish15 I meant that if you interpolate names like grid-col-{variable} tailwind JIT will not work. Of course if we bypass that and go with :style, that's fine. Will do that I think, this honestly just sucks

Ok, maybe we can try with this solution? https://stackoverflow.com/questions/67289894/jit-tailwindcss-using-variable-in-bg-not-rendering-color

Reply:
the only solution is to use `:style` binding, if we want to avoid the static classes in this situation